### PR TITLE
Document new keyboard shortcuts for Actions

### PR DESCRIPTION
**Pull request by konradpabjan**
### Why:

See recent changes in: https://github.com/github/github/pull/166218

There are new keyboard shortcuts for Actions!

### What's being changed:

If you click `shift ?` while on any Actions page you will see some new shortcuts, we need to document them. There are shortcuts for fullscreen logs, timestamps, and to quickly view the workflow file.

image

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/getting-started-with-github/keyboard-shortcuts.md [hunk 1]
```diff
@@ -159,6 +159,10 @@ For more keyboard shortcuts, see the [CodeMirror documentation](https://codemirr
 | Keyboard shortcut | Description
 |-----------|------------
 |<kbd>command + space </kbd> or <kbd>control + space</kbd> | In the workflow editor, get suggestions for your workflow file.
+|<kbd>g</kbd> <kbd>f</kbd> | Go to the workflow file
+|<kbd>shift + t</kbd> or <kbd>T</kbd> | Toggle timestamps in logs
+|<kbd>shift + f</kbd> or <kbd>F</kbd> | Toggle fullscreen logs
```
Comment (Legionsixxx9) [inline content/github/getting-started-with-github/keyboard-shortcuts.md:164]: 164

**Review by Legionsixxx9 — APPROVED** [review]
``````

**Comment by janiceilene** [comment]
Thanks for opening a PR @konradpabjan! I'll triage this for review ⚡ there may be a bit of a wait since lots of folks are out for the holidays 💖

**Review by Antoniorozy — APPROVED** [review]

**Comment by konradpabjan** [comment]
Verified my changes in Heroku, should be good to go! See: https://docs-2227--konradpabjanactions.herokuapp.com/en/free-pro-team@latest/github/getting-started-with-github/keyboard-shortcuts#github-actions

image

**Review by skedwards88 — APPROVED** [review]
@konradpabjan Thank you for documenting these new shortcuts! I'll get this merged for you.

**Comment by skedwards88** [comment]
Update: I'll get this merged once the repo unfreezes.

**Pull request merged**